src/index.ts

```diff
@@ -1,17 +1,34 @@
 import { VoiceChannel } from 'discord.js';
 import { createVoiceConnection } from './VoiceConnection';
 import { JoinConfig } from './DataStore';
+import generateDependencyReport from './util/generateDependencyReport';
+
+/**
+ * The options that can be given when joining a voice channel
+ */
+export interface JoinVoiceChannelOptions {
+	/**
+	 * If true, debug messages will be enabled for the voice connection and its
+	 * related components. Defaults to true.
```

Comment:
```suggestion
	 * related components. Defaults to false.
```

Although this isn't used in the rest of the library, this is supported by tsdocs:

```suggestion
	 * related components.
	 * @default false
```

If we go for this style, it should then be applied everywhere else.